wrappers/http/client.go

```diff
@@ -176,7 +176,29 @@ func getOperationName(url string) string {
 	if err != nil {
 		return ""
 	}
-	return parsedURL.Host + parsedURL.Path
+	return parsedURL.Host + getNormalizedPath(parsedURL.Path)
+}
+
+func getNormalizedPath(urlPath string) string {
+	depth := config.HTTPIntegrationUrlPathDepth
```

Comment:
Are we sure that protocol part (`"http://` or `"https://`) is not exist in the `urlPath`? If not, can we trim it here?

Reply:
Yes, we are sure schme part is not in urlPath